pennylane/ops/op_math/__init__.py

```diff
@@ -105,4 +120,14 @@
 from .symbolicop import ScalarSymbolicOp, SymbolicOp
 from .controlled_decompositions import ctrl_decomp_zyz, ctrl_decomp_bisect
 
-controlled_qubit_ops = {"ControlledQubitUnitary", "CY", "CZ"}
+controlled_qubit_ops = {
```

Comment:
why does this set exist?

Reply:
It's used in `ops/__init__.py` with a comment that mentions something about avoiding circular imports